src/platforms/javascript/common/session-replay/troubleshooting.mdx

```diff
@@ -44,6 +44,22 @@ Another reason you might not see the body is if the format is not supported. We
 
 More details about this feature can be found in the <PlatformLink to="/session-replay/configuration/">configuration page</PlatformLink>.
 
+## A large number of mutations was detected ( N ). This can slow down the Replay SDK and impact your customers.
+
+The Sentry SDK attempts to minimize potential [performance overhead in a few different ways](/product/session-replay/performance-overhead/#how-is-session-replay-optimized).
+One such way is by keeping track of how many DOM mutations are happening. This is important because many mutations can slow down the web page. And for Replay to keep track of it,
+it could inccur an additional slow down. This is an indication your page could be optimized. For example, a drop down list with thousands of entries could be refactored into one with lazy loading.
+Fetching more data and updating the drop down list as the user scroll through it.
+The SDK has a <PlatformLink to="/session-replay/configuration/#mutation-limits">configuration that allows you configure the limits</PlatformLink> before recordings stops.
+
+## The SDK is slowing down my website
+
+Sentry's SDK latest version includes several performance improvements. Two of the major culprits for performance impact we've identified
+where due to websites having extremely large number of mutations happening (see above _large number of mutations_), and very large console messages. We've added improvements for these
+cases such as [truncating the console](https://github.com/getsentry/sentry-javascript/pull/7917) and disabling Replay recording on pages that have excessively high number of mutations.
+It's important to update the SDK to the latest version to benefit from these improvements. In the event you encounter performance problems with Session Replay on the latest version of the SDK,
+please [open a GitHub issue](https://github.com/getsentry/sentry-javascript/issues/new?assignees=&labels=Type%3A+Bug&projects=&template=bug.yml) describing the situation so we can further investigate.
+
```

Comment:
```suggestion
## The SDK is slowing down my website

Sentry's ReplaySDK v??? and above includes several performance improvements. Two of the major culprits for performance impact we've identified are due to websites having extremely large number of mutations happening (see above _large number of mutations_), and very large console messages. We've added improvements for these cases such as [truncating the console](https://github.com/getsentry/sentry-javascript/pull/7917) and disabling Replay recording on pages that have excessively high number of mutations. It's important to update the SDK to the latest version to benefit from these improvements. In the event you encounter performance problems with Session Replay on the latest version of the SDK, please [open a GitHub issue](https://github.com/getsentry/sentry-javascript/issues/new?assignees=&labels=Type%3A+Bug&projects=&template=bug.yml) describing the situation so we can further investigate.

```

Reply:
I edited ur suggestion to remove the reference to the exact version (XXX) in favor of leaving it open again. I understand this isn't exact, but I'm purposely trying to just say "the latest" to hint nudge folks into updating and verifying whether it's slow still, that's because we continue to add more perf improvements constantly